app/models/hmcts_common_platform/prosecution_case_summary.rb

```diff
@@ -26,6 +26,12 @@ def hearing_summaries
       end
     end
 
+    def application_summaries
```

Comment:
this should be singular `ApplicationSummary`

Reply:
I've reconsidered my previous comment.
Since application_summaries returns an array, it makes sense for the name to be plural.

Even though the property inside the JSON is singular (applicationSummary), it's consistent with how defendant_summaries and hearing_summaries are handled.

So, I'd suggest we stick with the existing convention for now.